ci/linux_apisix_current_luarocks_runner.sh

```diff
@@ -66,6 +67,10 @@ script() {
     ulimit -n -S
     ulimit -n -H
 
+    # set default admin token
+    sudo yamlq e -i '.apisix.admin_key[0].key = "edd1c9f034335f136f87ad84b625c8f1"' conf/config-default.yaml
```

Comment:
why set token twice https://github.com/apache/apisix/pull/6214/files#diff-210dceffdb6579327475828287fd91c559b209de07ebf0323ab8005b589d9c0fR44？

Reply:
There are two role users in the default configuration template of APISIX, namely `admin` and `viewer`.